web/src/components/Wrapper.js

```diff
@@ -22,11 +22,14 @@ class Wrapper extends Component {
     const showSiteTitle = document.location.pathname === '/';
     return (
       <div className="site">
+        <a href="#apd-title" className="skip-nav">
```

Comment:
This ID looks like it only exists on the APD page itself, but the "Skip to main content" link is on every page with a header (so login, admin pages, etc.) Does it make sense to instead give the main `h1` on every page the same ID? Or maybe the container that wraps our main content on every page?

Reply:
![smart](https://media.giphy.com/media/hFROvOhBPQVRm/giphy.gif)